BoxContentSDK/BoxContentSDK/BOXContentSDKErrors.h

```diff
@@ -84,3 +87,5 @@ extern NSString *const BOXAuthErrorMissingDeviceId;
 extern NSString *const BOXAuthErrorUnsupportedDevicePinningRuntime;
 
 extern NSString *const BOXAuthErrorAccessDenied; // User denied access to their Box account
+
+extern NSString *const BOXRequestErrorRecentSimilarComment;
```

Comment:
Is this unique to comments or can the error be more generic like identicalRequest or duplicateRequest or something so that it could later apply to other things?

Or even, what is the error for adding a collaborator who is already collaborated? Is that appropriate for this type of situation? Adding a comment that was already added...